### DISCUSSION TITLE
beta prefix

### DISCUSSION THREAD

**Discussion by dnyandeobharambe** [GitTools/GitVersion#3862] [Q&A]
I am using gitversion in azure devops pipeline. I am using version 5.12.0 and 5.11.1 in both case beta prefix added in my version .
I get version values as 1.1.0-beta.219  while I am interested in clean version like 1.1.1 or similar. Can somebody explain me how I get it. 
My GitVersion.yml looks like this 
mode: Mainline
major-version-bump-message: '\+semver:\s?(breaking|major)'
minor-version-bump-message: '\+semver:\s?(feature|minor)'
patch-version-bump-message: '\+semver:\s?(fix|patch)'
no-bump-message: '\+semver:\s?(none|skip)'

If I add tag:' ' then it says tag is not property available. I stuck due to this . Can somebody guide me?

**Comment by asbjornu** [comment]
You get a "clean" (stable) version by applying `git tag 1.1.1` to the appropriate commit.
